docs/content/topics/developers.md

```diff
@@ -108,297 +114,232 @@ seamlessly with Docker Desktop (Docker for Windows).
 To run lint checks:
 
 ```console
-$ make lint
-```
-
-To format the Go files:
-
-```console
-$ make format-go
+$ make lint-go
 ```
 
 To run the unit tests:
 
 ```console
-$ make test-unit
-```
-
-To re-run Go dependency resolution:
-
-```console
-$ make dep
+$ make test-unit-go
 ```
-
 ## Working with JS Code (for the Brigade Worker)
 
-To format the Javascript files:
+To lint the Javascript files:
 
 ```console
-$ make format-js
+$ make lint-js
 ```
 
 To run the tests:
 
 ```console
-$ make test-js
+$ make test-unit-js
 ```
 
-To re-run JS dependency resolution:
+To clear the JS dependency cache:
 
 ```console
-$ make yarn-install
+$ make clean-js
```

Comment:
I don't think we have such a target anymore.

Reply:
It appears we do... but were you thinking of mentioning a different one here?

Reply:
My bad. Yes. This still exists.